### ISSUE TITLE
Panic on "Build constraints exclude all Go files"

### ISSUE THREAD

**Issue by dustin-ward** [ZOSOpenTools/wharf#22]
When wharf encounters a package that contains no "buildable" Go files a panic occurs:

[github.com/elastic/beats/filebeat](https://github.com/elastic/beats) as an example:
```
unable to load packages: unable to load github.com/elastic/beats/v7/libbeat/processors/add_docker_metadata: build constraints exclude all Go files in /u/kishku1/projects/wharf-work/beats/libbeat/processors/add_docker_metadata
```

i.e. All files within this package are tagged in a way that zOS is excluded. The best solution for most cases is to simply apply the `zos` tag to each file within the package.

**Comment by dustin-ward** [comment]
A fix is in progress

**Comment by bluecivet** [comment]
fix in 65a96663361795d30
